### DISCUSSION TITLE
NEED HELP :(

### DISCUSSION THREAD

**Discussion by cevallos98** [jilleb/MQB-FPA#63] [Q&A] [answered]
so here is the problem. I updated the gateway of my leon cupra 5F, I used the gateway with AA termination
A friend put a dataset of a 2020 leon but the door light change does not work when changing driving mode. I know a bit about programming but I'm not an expert.
Could someone give me a guide on how to create a correct dataset for my car?

**Comment by olli991** [comment]
Does you car only have red and white or do you have RGB lights in the doors. 

The change with profile was an old behavior which in stock is not present in the newer RGB style cars. You can code it back but you need a special FPA DS then as well.

**Comment by lito69** [comment, marked as answer]
> Así que aquí está el problema. Actualicé el gateway de mi leon cupra 5F, usé el gateway con terminación AA Un amigo puso un dataset de un leon 2020 pero no funciona el cambio de luz de puerta al cambiar de modo de conducción. Sé un poco de programación pero no soy un experto. ¿Alguien podría darme una guía sobre cómo crear un conjunto de datos correcto para mi automóvil?

We can help you, I have personally made datasets for cupra rgb or with red white led.
https://t.me/codingmqb

**Reply by olli991** [reply]
@lito69 kind reminder that we're not a plattform where you can SELL you services!

**Reply by lito69** [reply]
Lo
[19_3Q0907530AC_Leon_cupra_FPA_AMB only white red.zip](https://github.com/jilleb/MQB-FPA/files/9541466/19_3Q0907530AC_Leon_cupra_FPA_AMB.only.white.red.zip)
> @lito69amable recordatorio de que no somos una plataforma donde puedes VENDER tus servicios!

Don't worry, I do things for free just like you, forever... I think you know.

**Reply by cevallos98** [reply]
we did it! We repair the lighting function of the doors thanks to @lito69, thanks to his dataset and his knowledge in coding. he does not charge anything. thanks for the reply @olli991.

**Reply by olli991** [reply]
@cevallos98 please acceppt some answer then to close this discussion 😄
